dagger-tests/src/integrationtest/java/io/odpf/dagger/integrationtest/HttpExternalPostProcessorIntegrationTest.java

```diff
@@ -1,7 +1,8 @@
 package io.odpf.dagger.integrationtest;
 
+import io.odpf.dagger.common.core.DaggerContextTestBase;
 import org.apache.flink.api.java.utils.ParameterTool;
-import org.apache.flink.runtime.testutils.MiniClusterResourceConfiguration;
+//import org.apache.flink.runtime.testutils.MiniClusterResourceConfiguration;
```

Comment:
Any reason this import been commented ? 
I can see its still used in Line 50 below

Reply:
This has been un-commented and handled.